roles/keystone/tasks/main.yml

```diff
@@ -81,7 +81,7 @@
             owner=keystone group=keystone
   with_fileglob: ../templates/etc/keystone/*
   notify:
-    - restart keystone services
```

Comment:
This is correct. The keystone service is not managed by apache. Keystone is configure  using uwsgi and you should be able to restart using `service keystone restart`